runtime/lib/capi/RuntimeCAPI.cpp

```diff
@@ -59,7 +59,7 @@ void *_mlir_memref_to_llvm_aligned_alloc(size_t alignment, size_t size)
 void _mlir_memref_to_llvm_free(void *ptr)
 {
     Catalyst::Runtime::CAPI::DRIVER->get_memory_manager()->erase(ptr);
-    free(ptr);
+    /* free(ptr); */
```

Comment:
This can change can be reverted as well.

Reply:
OK, I need to check it against the 121

Reply:
Now it is reverted